docs/antrea-ipam.md

```diff
@@ -193,18 +194,26 @@ restore its configurations at exit. Node may lose network connection when `antre
 or OVS daemons are stopped unexpectedly, which can be recovered by rebooting the Node.
 `AntreaIPAM` Pods' traffic will not be routed by local Node's network stack.
 
-All traffic to a local Pod will be sent to the Pod's OVS port directly, after the
-destination MAC is rewritten to the Pod's MAC address. This includes `AntreaIPAM` Pods
-and regular `Subnet per Node` IPAM Pods, even they are not in the same subnets.
-Inter-Node traffic will be sent to the Node network from the source Node, and forwarded
-to the destination Node by the Node network.
+Traffic from `AntreaIPAM` Pods without VLAN, regular `Subnet per Node` IPAM Pods, and K8s
+Nodes are recognized as VLAN 0 (untagged).
+
+All traffic to a local Pod in same VLAN will be sent to the Pod's OVS port directly,
+after the destination MAC is rewritten to the Pod's MAC address. This includes
+`AntreaIPAM` Pods and regular `Subnet per Node` IPAM Pods, even when they are not in the
+same subnet.
+All traffic to a Pod in different VLAN will be sent to the gateway of underlay network.
```

Comment:
Do you want to start a new paragraph for it? If yes, an empty line is needed before it to indicate a new paragraph in markdown.  So do the following two cases.

Reply:
No, this is to make the raw text clear, not to start a new paragraph.

Reply:
I would say it is even confusing. If you want not a new paragraph for readers, why start a new in "raw text"?

Reply:
@jianjuns Already changed to one line according your suggestion.
If we start a new line for every sentence, we don't need to correct the layout for the whole paragraph when we add/remove some words before this sentence in this paragraph.

Reply:
Many IDE/editor can wrap lines automatically. A line for a sentence doesn't look tidy and makes "paragraph" meaningless, and it's not a convention in practice in any project I know.

Reply:
OK, got it.